src/seedu/addressbook/data/tag/UniqueTagList.java

```diff
@@ -94,18 +94,6 @@ public boolean contains(Tag toCheck) {
     }
 
     /**
-     * Adds a Tag to the list.
-     *
-     * @throws DuplicateTagException if the Tag to add is a duplicate of an existing Tag in the list.
-     */
-    public void add(Tag toAdd) throws DuplicateTagException {
```

Comment:
Nice spot, but this should probably be in a separate commit because [v1 1/3] is about `AddressBook#addTag()`, not `UniqueTagList#add()`.

Also, not sure whether it is a good reason to remove `UniqueTagList#add()` just because `AddressBook#addTag()` no longer exist, since I can understand that you are not supposed to manipulate `AddressBook`'s tags directly, but I don't expect `UniqueTagList` to no longer allow adding new tags as a result. @chao1995 Your thoughts on this?